ui.py

```diff
@@ -19,7 +19,9 @@ def display_menu_get_choice(menu):
 def message(msg):
     """ Prints a message for the user
      :param msg: the message to print"""
+    print(len(msg)*'*')
```

Comment:
This isn't specifically related to the goodbye message. if the messages need stars before and after, file a new issue and make the change in a new PR